frontend/config-assistant/src/components/toolbar/TopToolbar.vue

```diff
@@ -285,9 +324,21 @@ watchDebounced(
 </script>
 
 <template>
-  <Dialog v-model:visible="topMenuBar.showDialog.value">
+  <Dialog v-model:visible="showDialog" header="Select a Schema">
```

Comment:
Can you please make a new component "InitialSchemaSelectionDialog" out of this? 
The top toolbar file is already way to big and hard to understand. You can use the emits to emit a selection event, depending on which the corresponding functionality is called. you can also defineExpose() to have a "show" method to be called from outside (see the SchemInfoOverlayPanel, there I used this)

Reply:
If you add a new component for this feature, a good place to import and use this component would be inside the FileEditorView.vue or CombinedEditorComponent.vue.